index.bs

```diff
@@ -96,6 +96,40 @@ run the following steps:
 
 </div>
 
+## Permissions ## {#permissions}
+
+Note: By default entries inherit their permissions from their parent. If an entry doesn't
+have a parent, permissions are by default granted, and requesting permissions is a no-op.
+Entries returned by {{chooseFileSystemEntries()}} override these steps, providing stricter
+access control for those entries.
+
+Issue: Figure out some way to integrate this better with the permissions API [[permissions]],
+for example by making the entry be a part of the PermissionDescriptor.
+
+<div algorithm>
+An [=/entry=] |entry| has associated <dfn for=entry>query permission steps</dfn>,
+which take a {{FileSystemHandlePermissionDescriptor}} and return a {{PermissionState}}.
+Unless specified otherwise, these steps are:
+1. Let |parent| be a [=directory entry=] where |parent|.[=children=] [=list/contains=] |entry|,
```

Comment:
This is a little awkward. 

How about defining "an entry's parent" somewhere? 

Also, is "a directory entry" intentionally indefinite? i.e. can there be more than one directory that contains the same entry?

Reply:
Thanks for reviewing this! I agree with it being awkward. I rephrased things a bit as you suggested, hopefully being clearer?

Reply:
Yep! And sorry for the single comment, I meant to batch up more...